services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/model/OfInput.java

```diff
@@ -63,13 +64,20 @@ public U64 packetInCookie() {
             return null;
         }
 
-        if (packetInReservedCookieValue.equals(cookie)) {
-            return null;
-        }
-
         return cookie;
     }
 
+    /**
+     * Check current packet(PACKET_IN) cookie value is mismatched with expected value.
+     */
+    public boolean packetInCookieMismatch(U64 expected, Logger log) {
```

Comment:
1. It's not obvious why the method accept the logger.
2. Should the getter method decide whether to log the result or not?

I think the method "unwrap" (which calls "packetInCookieMismatch") can log the details by itself.

Reply:
The only reason to log "match" results here - is complex(relatively) log message and several(2 at this moment) usage of this check. To keep ability to distinguish exact caller - passed logger instance is used to log messages.